packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts

```diff
@@ -9,7 +9,7 @@ export interface UseFeedInfiniteScrollProps {
 
 export const InfiniteScrollScreenOffset = classed(
   'div',
-  'absolute bottom-screen desktopL:bottom-screen-20 left-0 w-px h-px opacity-0 pointer-events-none',
+  'bottom-screen desktopL:bottom-screen-20 left-0 w-px h-px opacity-0 pointer-events-none',
```

Comment:
Was removal of the `absolute` class here intentional? If yes, why do we still need `left-0`?

Reply:
Yeah, it was intentional.
Probably don't need `left-0`. Will double check.

Reply:
```suggestion
  'bottom-screen desktopL:bottom-screen-20 w-px h-px opacity-0 pointer-events-none',
```

Reply:
To add to that, without absolute `bottom-screen` also doesn't get added, so might want to clean it up.
![Screenshot 2024-01-15 at 13 40 18](https://github.com/dailydotdev/apps/assets/554874/d43b9091-fa52-45b4-b4fd-b725144b2120)

Reply:
🙈